visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -131,7 +131,7 @@ MotionPlanningDisplay::MotionPlanningDisplay() :
   show_workspace_property_ = new rviz::BoolProperty("Show Workspace", false, "Shows the axis-aligned bounding box for the workspace allowed for planning",
                                                     plan_category_,
                                                     SLOT(changedWorkspace()), this);
-  query_start_state_property_ = new rviz::BoolProperty("Query Start State", true, "Shows the start state for the motion planning query",
+  query_start_state_property_ = new rviz::BoolProperty("Query Start State", false, "Set a custom start state for the motion planning query",
```

Comment:
Valid point. I agree that it might be reasonable to disable the StartState marker by default.
However, associating visibility of the StartState marker with enabling of execution isn't convincing.